android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -38,52 +38,51 @@
 
 public class EditorFragment extends BaseMwmFragment implements View.OnClickListener, EditTextDialogFragment.OnTextSaveListener
 {
-  final static String LAST_LOCALIZED_NAME_INDEX = "LastLocalizedNameIndex";
+  final static String LAST_INDEX_OF_NAMES_ARRAY = "LastIndexOfNamesArray";
 
   private TextView mCategory;
-  private View mCardName;
   private View mCardAddress;
   private View mCardMetadata;
-  private EditText mName;
 
-  private RecyclerView mLocalizedNames;
+  private RecyclerView mNamesView;
 
-  private final RecyclerView.AdapterDataObserver mLocalizedNamesObserver = new RecyclerView.AdapterDataObserver()
+  private final RecyclerView.AdapterDataObserver mNamesObserver = new RecyclerView.AdapterDataObserver()
   {
     @Override
     public void onChanged()
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeChanged(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeInserted(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeRemoved(int positionStart, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
 
     @Override
     public void onItemRangeMoved(int fromPosition, int toPosition, int itemCount)
     {
-      refreshLocalizedNames();
+      refreshNamesCaption();
     }
   };
 
-  private MultilanguageAdapter mLocalizedNamesAdapter;
-  private TextView mLocalizedShow;
-  private boolean mIsLocalizedShown;
+  private MultilanguageAdapter mNamesAdapter;
+  private TextView mNamesCaption;
+  private TextView  mAddLanguage;
```

Comment:
BTW, maybe it worth to set up clang-format for java too? (not in this CL, of course)